JSONAPI/Http/QueryableToManyRelatedResourceDocumentMaterializer.cs

```diff
@@ -46,13 +46,5 @@ public abstract class QueryableToManyRelatedResourceDocumentMaterializer<TRelate
         /// </summary>
         protected abstract Task<IQueryable<TRelated>> GetRelatedQuery(string primaryResourceId, CancellationToken cancellationToken);
 
-        /// <summary>
-        /// If the client doesn't request any sort expressions, these expressions will be used for sorting instead.
-        /// </summary>
-        /// <returns></returns>
-        protected virtual string[] GetDefaultSortExpressions()
```

Comment:
If you want to sub-class `QueryableToManyRelatedResourceDocumentMaterializer` and provide default sort expressions for a particular endpoint in the case the client does not provide their own sort expressions, this would be the only way to do it. I was doing precisely this in my own project at one point, which is why I added the method.(although I've since switched to my own custom implementation of IRelatedResourceDocumentMaterializer).